scripts/actors-models-graph.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python
```

Comment:
ha. this will be the problem.

```suggestion
#!/usr/bin/python3
```

Reply:
Well, this would break it for python2 setups...

Reply:
Do we care about python2 really? Python2 leapp libs are even not built for Fedora and in case it will be part of snactor, shebang is not relevant. so I would go with Python3 - as Python2 will be death soon (https://pythonclock.org/)

Reply:
We do, because of RHEL7 at this moment